Update sign out behavior and release notes (#105)
Esri/data-collection-dotnet: RELEASE.md

    * Adds a `ModernMapPanel` custom layout panel to facilitate consistent, responsive design for UWP and WPF.
    * Refactors styles and related XAML.
    * Switches to vector icons where possible.
* Moves the .NET Core WPF project to a separate folder to prevent possible build issues.

> * Updates the code called at sign out to revoke credentials in addition to deleting them locally, when possible.

Reviewer: ```suggestion
* Updates the code called at sign out to revoke credentials from the server, when possible, and delete them locally.
```
